common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -333,6 +334,28 @@ def selected_children(self):
 
         return self.selected
 
+    @XBlock.handler
+    def reset_selected_children(self, context):
+        """
+        This is a handler that deletes a user state of selected children to be displayed, 
+        and makes a new selection of children to be display to the current user.
+
+        This reads and empties the "selected" field, which has a user_state scope,
+        then calls the student view method which initiates the process of making 
+        a new selection.
+        """
+
+        # re initialize the self.selected field. not sure if the selelcted field value is saved
+        # to the runtime kvstore or something similar
+        self.selected = List(
+            # This is a list of (block_type, block_id) tuples used to record
+            # which random/first set of matching blocks was selected per user
+            default=[],
+            scope=Scope.user_state,
+        )
```

Comment:
Wouldn't `self.selected = []` work here?

Reply:
I wasn't sure about using a python list directly since it was previously initialised by the xblock field `List`. But I've found out that it works